src/deb/init.d/jmxtrans

```diff
@@ -1,30 +1,30 @@
 #!/bin/bash
```

Comment:
Keeping the file mode to `0755` make sense. It is an executable script, we should version it as such (yes, I know, file mode will be modified by `dpkg` on installation).